modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/api.ts

```diff
@@ -459,3 +411,51 @@ export async function putObjectRelationship({
 		url: `/o/object-admin/v1.0/object-relationships/${id}`,
 	});
 }
+
+export async function save({
```

Comment:
@MarinhoFeliphe @igor-franca the save function is used in other functions of this file, that's why it was being declared before to respect precedence order. Should we keep it this way?

Reply:
Hey @naraandrade, I sorted the entire file alphabetically, this is the approach we used on backend side,

the old approach suggested by Brian is bad, I can talk to him to share my motivation and share a lot of places ([this one for example](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-admin-rest-impl/rest-openapi.yaml)) that we use the same approach on Backend side.

As soon as we check the changes are not breaking anything Me and @igor-franca can review this pull request along side with Brian.

Reply:
Ok, thanks!